src/models/cart.dao.js

```diff
@@ -0,0 +1,65 @@
+const { appDataSource } = require("./data-source");
+
+const createCart = async (quantity, userId, productOptionId) => {
+  await appDataSource.query(
+    `
+    INSERT INTO carts (
+      quantity,
+      user_id,
+      product_option_id
+    ) VALUES (
+      ?,
+      ?,
+      ?
+    );
+    `,
+    [quantity, userId, productOptionId]
+  );
+};
+
+const getCart = async (userId) => {
```

Comment:
```suggestion
const getCarts = async (userId) => {
```
naming을 명확히 하자면 해당 query는 Array(`[]`), 즉 배열을 반환할 것이기 때문에 복수를 나타내는 변수명으로 지어주시는 것이 보다 더욱 정확합니다.

Reply:
넵 알겠습니다. 다른 변수명도 모두 바꾸겠습니다!